schema/codelists/language.csv

```diff
@@ -0,0 +1,185 @@
+Code,Title
+aa,Afar
+ab,Abkhazian
+ae,Avestan
+af,Afrikaans
+ak,Akan
+am,Amharic
+an,Aragonese
+ar,Arabic
+as,Assamese
+av,Avaric
+ay,Aymara
+az,Azerbaijani
+ba,Bashkir
+be,Belarusian
+bg,Bulgarian
+bh,Bihari languages
+bi,Bislama
+bm,Bambara
+bn,Bengali
+bo,Tibetan
+br,Breton
+bs,Bosnian
+ca,"Catalan, Valencian"
+ce,Chechen
+ch,Chamorro
+co,Corsican
+cr,Cree
+cs,Czech
+cu,"Church Slavic, Old Slavonic, Church Slavonic, Old Bulgarian, Old Church Slavonic"
+cv,Chuvash
+cy,Welsh
+da,Danish
+de,German
+dv,"Divehi, Dhivehi, Maldivian"
+dz,Dzongkha
+ee,Ewe
+el,Modern (1453-) Greek
```

Comment:
Oops, my change shouldn't apply to this line (and maybe not to "Sotho, Southern"). I'll commit a fix.

Reply:
Looking at Wikipedia, "Southern Sotho" and "Modern Greek" are correct. I've made another commit to remove parentheses, which don't disambiguate (at least in ISO639-1 - maybe they have a function in the other ISO639 standards).